test/integration-tests-1.7.7.x/cypress/integration/PageObjects/objects.js

```diff
@@ -0,0 +1,150 @@
+        cy.get('#cardholderNameInput').type('testname')
+        cy.get('#pensioCreditCardPaymentSubmitButton').click().wait(4000)
+
+    }
+
+    klarna_payment(){
+
+        cy.get(':nth-child(2) > .col-xs-12 > .payment_module > .altapay').click().wait(1000)
+        cy.get('[id=submitbutton]').click().wait(3000)
+        cy.get('[id=klarna-pay-later-fullscreen]').wait(5000).then(function($iFrame){
+            const mobileNum = $iFrame.contents().find('[id=invoice_kp-purchase-approval-form-phone-number]')
+            cy.wrap(mobileNum).type('(452) 012-3456')
+            const personalNum = $iFrame.contents().find('[id=invoice_kp-purchase-approval-form-national-identification-number]')
+            cy.wrap(personalNum).type('1012201234')
+            const submit = $iFrame.contents().find('[id=invoice_kp-purchase-approval-form-continue-button]')
+            cy.wrap(submit).click().wait(4000)
+            
+        })
+        
+    }
+
+    admin()
+    {
+            cy.clearCookies()
+            cy.fixture('config').then((admin)=>{
+            cy.visit(admin.adminURL)
+            cy.get('#email').type(admin.adminUsername)
+            cy.get('#passwd').type(admin.adminPass).wait(2000)
+            cy.get('.ladda-label').click().wait(3000)
+            cy.visit(admin.adminURL).wait(3000)
+            })
+
+    }
+
+    capture(){
+
+        // 1.6.X
+        // cy.get('#maintab-AdminParentOrders > .title').click()
+        
+        // cy.get('tbody > :nth-child(1) > .fixed-width-xs').click().wait(1000)
+        // //Capture
+        // cy.get('[id=transactionOptions]').then(function($iFrame){
+        //     const capture = $iFrame.contents().find('[id=btn-capture]')
+        //     cy.wrap(capture).click({force: true})
+        //     cy.get('#popup_ok').click()
+        //     cy.get('#popup_ok').click()
+
+        
+            
+            
+//         })
+//         cy.get('#altapay > div > div > div.row.panel-body > div:nth-child(4) > div:nth-child(1) > div > div > table > tbody > tr:nth-child(1) > td:nth-child(2)').should('have.text', 'captured')
+        
+//         //Refund
+//         cy.get('[id=transactionOptions]').then(function($iFrame){
+//             const capture = $iFrame.contents().find('[id=btn-refund]')
+//             cy.wrap(capture).click({force: true})
+//             cy.get(':nth-child(2) > :nth-child(10) > .form-control').type('3').click()
+            
+//         })
+            
+//         cy.get('[id=transactionOptions]').then(function($iFrame){
+//             const capture = $iFrame.contents().find('[id=btn-refund]')
+//             cy.wrap(capture).click({force: true})
+//             cy.get('#popup_ok').click()
+//             cy.get('#popup_ok').click()
+//         })
+//         cy.get('#altapay > div > div > div.row.panel-body > div:nth-child(4) > div:nth-child(1) > div > div > table > tbody > tr:nth-child(1) > td:nth-child(2)').should('have.text', 'refunded')
+        
+//     }
```

Comment:
Any reason why we are leaving this commented out code?

Reply:
As currently our plugin is not ready for 1.7.7.x that's why these tests are in pause state. Actually these tests are written in advance and are almost ready but and not sure how smooth they will run. So when our plugin gets ready for 1.7.7.x then instead of writing tests time we will just verify and make the necessary changes if required. So it will take less time in near future.